src/main/kotlin/no/nav/familie/ba/sak/common/RessursUtils.kt

```diff
@@ -100,6 +100,7 @@ object RessursUtils {
             if (funksjonellFeil.throwable != null) "[${funksjonellFeil.throwable!!::class.java.name}] " else ""
 
         logger.info("$className En funksjonell feil har oppstått(${funksjonellFeil.httpStatus}): ${funksjonellFeil.message} ")
```

Comment:
Kan vurdere å legge til denne biten også 

```suggestion
        logger.info("$className En funksjonell feil har oppstått(${funksjonellFeil.httpStatus}): ${funksjonellFeil.message}. Se Securelogs for detaljer.")
```